### PR TITLE
Fix redirects

### PR DESCRIPTION
**Pull request by matteopilz**
#### Brief description of what is fixed or changed

I don't know what happened to FLASHIda, did we remove that from the applications page?

### CODE REVIEW

**Comment by jpfeuffer** [comment]
Probably there was never a separate page for it. Just an overview on the application overview page. But that is not how it was intended, so I changed it.
You could copy the contents of the FlashIDA section of the applications.md.bup file into a separate flashida.md file in the application folder.

File: content/en/applications/flashida.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: FLASHIca
```
Comment (jpfeuffer) [inline content/en/applications/flashida.md:2]:
```suggestion
title: FLASHIda
```

File: static/_redirects [hunk 2]
```diff
@@ -31,13 +31,40 @@
...
+/nase /applications/nase 200
+/NASE /applications/nase 200
+
+/openpepxl /applications/openpepxl 200
+/OpenpepXL /applications/openpepxl 200
+
+/rnpxl /applications/rnpxl 200
+/RNPxl /applications/rnpxl 200
+
 /FLASHIda /applications/#intelligent-data-acquisition-for-top-down-proteomics-flashida 200
```
Comment (jpfeuffer) [inline static/_redirects:64]: This line and the following must be changed, too.

**Review by jpfeuffer — CHANGES_REQUESTED** [review]

File: static/_redirects (hunk 2 above)
Comment (matteopilz) [reply static/_redirects:64]: To what?

Comment (jpfeuffer) [reply static/_redirects:64]: /applications/nameofyournewfile.md

I guess /applications/flashida

**Comment by cbielow** [comment]
ready for merge @matteopilz  @jpfeuffer ?

**Comment by jpfeuffer** [comment]
No, see unresolved comments.

**Review by jpfeuffer — APPROVED** [review]

**Pull request merged**